packages/build-electron/src/start/index.ts

```diff
@@ -8,27 +9,28 @@ import {
     BuildEvent
 } from '@angular-devkit/architect';
 import { normalize, getSystemPath } from '@angular-devkit/core';
+import { WebpackBuilder } from '@angular-devkit/build-webpack';
 
 import { Observable, of, merge, TeardownLogic } from 'rxjs';
 import { concatMap, share, first, switchMap, dematerialize, materialize } from 'rxjs/operators';
 
 import { ElectronStartBuilderSchema } from './schema';
-import { WebpackBuilder } from '@angular-devkit/build-webpack';
 
 export class ElectronStartBuilder implements Builder<ElectronStartBuilderSchema> {
 
     constructor(public context: BuilderContext) { }
 
     run(builderConfig: BuilderConfiguration<ElectronStartBuilderSchema>): Observable<BuildEvent> {
-        const targetString = builderConfig.options.browserTarget;
-        const [project, target, configuration] = targetString.split(':');
+        const browserTarget = builderConfig.options.browserTarget.replace(':serve', ':build');
```

Comment:
Why do you need to change the serve to build, does it mean there will be no watch on the angular project files?